File: fastapi_lotto_result/app/data/model/lotto_draw_result.py
from sqlalchemy import Column, Integer, String , BigInteger
from app.data.model.db_base_model import DbModelBase

class LottoDrawResult(DbModelBase):
    __tablename__ = 'lotto_draw_result'
    drw_no = Column(Integer, primary_key=True)
    drw_no_date = Column(String)
    drwt_no1 = Column(Integer)
    drwt_no2 = Column(Integer)
    drwt_no3 = Column(Integer)
    drwt_no4 = Column(Integer)
    drwt_no5 = Column(Integer)
    drwt_no6 = Column(Integer)
    bnus_no = Column(Integer)
    tot_sell_amnt = Column(BigInteger)
    first_win_amnt = Column(BigInteger)
    first_prz_wner_co = Column(Integer)
    first_accum_amnt = Column(BigInteger)
    return_value = Column(String)

File: nuxt_vue3_lotto/app/data/model/lottoDrawResult.ts
export interface LottoDrawResult {
    returnValue: string; // 요청결과
    drwNoDate: string; // 날짜
    totSellamnt: number;
    firstWinamnt: number; //1등 상금액
    firstPrzwnerCo: number; // 1등 당첨인원
    firstAccumamnt: number;
    drwtNo1: number; // 로또번호 1
    drwtNo2: number; // 로또번호 2
    drwtNo3: number; // 로또번호 3
    drwtNo4: number; // 로또번호 4
    drwtNo5: number; // 로또번호 5
    drwtNo6: number; // 로또번호 6
    bnusNo: number; // 로또 보너스 번호
    drwNo: number; // 로또회차
}
